some icp additon repo

hrusikesh8280 opened the pull request.

eherrerosj commented on data/ecosystems/i/internet-computer.toml:
```diff
@@ -21728,3 +21728,205 @@ url = "https://github.com/NnsDao/nnsdao-fe"
+url = "https://github.com/ayeritedw/App_Classroom"
+
+[[repo]]
+url = "https://github.com/DevKipesa/client"
```
how's this using ICP?

eherrerosj commented on data/ecosystems/i/internet-computer.toml:
```diff
@@ -21728,3 +21728,205 @@ url = "https://github.com/NnsDao/nnsdao-fe"
 url = "https://github.com/michi-labs/d-snap"
 
+[[repo]]
+url = "https://github.com/GabrielPlus/Delicious"
```
how's this using ICP?

eherrerosj reviewed (changes requested):
left a comments

hrusikesh8280 commented:
i have reviewed the repos again and removed suggested changes, thanks for giving it a thorough look. Hope this helps

eherrerosj commented on data/ecosystems/i/internet-computer.toml:
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
+url = "https://github.com/ChristianDelRalba/ProyectoICP"
+
+[[repo]]
+url = "https://github.com/Glinrear/Mavain/tree/main/identity_system/canisters/identity"
```
this repo URL format is incorrect

eherrerosj commented on data/ecosystems/i/internet-computer.toml:
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
+url = "https://github.com/KaleF07/recipe-book"
+
+[[repo]]
+url = "https://github.com/Hexdee/OpenBridge"
```
how's this repo using ICP?

eherrerosj commented on data/ecosystems/i/internet-computer.toml:
```diff
@@ -21726,7 +21726,161 @@ url = "https://github.com/NeutronStarDAO/Proton"
+url = "https://github.com/ArielRobotti/ICP_Freelancer"
+
+[[repo]]
+url = "https://github.com/SistemasTecTlaxiaco/cafeteriaMagnate1"
```
this one belongs to NEAR I believe

eherrerosj reviewed (changes requested):
left a few more comments. Can you please make sure that the rest of the URLs are well formatted and that are repos that are actually using ICP please?

hrusikesh8280 commented:
> left a few more comments. Can you please make sure that the rest of the URLs are well formatted and that are repos that are actually using ICP please?
@eherrerosj , I have checked and corrected the format and removed unnecessary repos.

jubos commented:
Manually merged here [e6e0949](https://github.com/electric-capital/crypto-ecosystems/commit/e6e094980cd7161915b9fff5186915a5278060ca)
